action.yml

```diff
@@ -4,7 +4,8 @@ description: Get the PR associated with the current commit.
 inputs:
   github-token:
     description: The GitHub token used to create an authenticated client.
-    required: true
+    required: false
+    default: ${{ github.token }}
```

Comment:
does this need testing? I tried using it in my project, and have seen this in many other projects, including [competition](https://github.com/jwalton/gh-find-current-pr/blob/e12d66bc9ecc4fdcde07b0f70a3cb68ce7e4d807/action.yml#L8) 😉

Reply:
yeah 😄 this looks very solid. no need for extra testing